content/summary-resource/index.md

```diff
@@ -0,0 +1,101 @@
+---
+rfc: 0014
+start_date: 2018-08-13
+pr:
+status: draft
+---
+
+# Summary resource
+
+## Summary
+
+This RFC proposes a summary resource to offer an overview of the current state
+of the register.
+
+Dependencies:
+
+* [RFC0013: Multihash](https://github.com/openregister/registers-rfcs/pull/26)
+
+
+## Motivation
+
+The current summary is exposed as the register resource (`GET /register`).
+The name of the resource is misleading as you don't get the register, only
+some metadata about it. Also, the current specification and implementation
+don't agree on what is the register resource.
+
+## Explanation
+
+The summary resource is a new resource that will replace the old register
+resource. The summary resource will not contain any information about fields
+as this will be addressed by the schema resource.
+
+The most noticeable change is the addition of the `hashing-algorithm` object
+describing the hashing algorithm used in the register to create hashes for
+items, entries and proofs.
+
+***
+### Endpoint
+
+```
+GET /summary
+```
+
+### Response attributes
```

Comment:
would we want to include the root hash in this response?

Reply:
Yes, good call!